ui/src/checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import CheckBoxIcon from "@mui/icons-material/CheckBox";
```

Comment:
Are there enough files here to start adding some directory structure? e.g. maybe a `components/` subdir?

Reply:
Yeah, I have been thinking it's getting to that point too. I'll look at a `components/` directory at minimum and possibly others.